packages/scandipwa/src/ui/Link/Link.component.tsx

```diff
@@ -57,66 +52,54 @@ export class LinkComponent extends PureComponent<LinkComponentProps> {
             isOpenInNewTab,
             children,
             to,
+            attr,
+            mix,
+            mods,
+            block,
+            isUnstyled,
             ...props
         } = this.props;
 
-        if (isOpenInNewTab) {
-            return (
-                // eslint-disable-next-line jsx-a11y/click-events-have-key-events
-                <a
-                  { ...props }
-                  onClick={ this.scrollToElement }
-                  href={ to as string }
-                  rel="noopener noreferrer"
-                  target="_blank"
-                >
-                    { children }
-                </a>
-            );
-        }
-
         return (
-            // eslint-disable-next-line jsx-a11y/click-events-have-key-events
             <a
               { ...props }
+              block={ block }
+              mods={ isUnstyled ? {} : mods }
+              mix={ mix }
+              { ...attr as LinkHTMLAttributes<HTMLElement> }
               onClick={ this.scrollToElement }
               href={ to as string }
+              rel={ isOpenInNewTab && 'noopener noreferrer' }
+              target={ isOpenInNewTab && '_blank' }
```

Comment:
Just guessing, if isOpenInNewTab will be false, won't it mean that attribute target will have value false?
Same for rel